### PR TITLE
feat: UI to change Slashtags Web Relay

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

- similar to Electrum config we now have Slashtags Web Relay config

### Linked Issues/Tasks

Add any links to GitHub issues or Asana tasks that are relevant to this pull request.

### Type of change

New feature

### Tests

- [x] Detox test
- [x] Unit test

### Screenshot / Video

image

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,15 +8,6 @@ import { gestureHandlerRootHOC } from 'react-native-gesture-handler';
...
 import { name as appName } from './app.json';
-import { __E2E__ } from './src/constants/env';
-
-if (!__E2E__) {
-	// Polyfill fetch streaming on iOS (Android is not supported yet)
-	// https://github.com/react-native-community/fetch/issues/13
-	polyfillEncoding();
-	polyfillReadableStream();
-	polyfillFetch();
-}
```
Comment (pwltr) [inline index.js:19]: Why remove this? It's breaking something?

File: src/screens/Settings/WebRelay/index.tsx [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+				title: t('slashtags:error_saving_profile'),
+				description: res.error.message,
+			});
+		});
+
+		updateSlashPayConfig2({});
+
+		// ignore "profile" here
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [updated, webRelayClient, myProfileUrl, slashtagsProfile, t]);
```
Comment (pwltr) [inline src/screens/Settings/WebRelay/index.tsx:122]: Is this effect needed? Seems to me this can just be put at the bottom of `connectAndSave`.

**Review by pwltr — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (limpbrains) [reply index.js:19]: Take a look a few lines up, it's repeated twice

File: src/screens/Settings/WebRelay/index.tsx (hunk 2 above)
Comment (limpbrains) [reply src/screens/Settings/WebRelay/index.tsx:122]: Here we use new instance of webRelayClient.

File: index.js (hunk 1 above)
Comment (pwltr) [reply index.js:19]: Ah yeah, strange, probably from a merge conflict.

**Review by pwltr — APPROVED** [review]

**Pull request merged**